src/api/app/controllers/webui/request_controller.rb

```diff
@@ -209,12 +209,7 @@ def accept_request
 
   def forward_request_to(fwd)
     tgt_prj, tgt_pkg = params[fwd].split('_#_') # split off 'forward_' and split into project and package
-    description = @req.description
-    description ||= ""
-    who = @req.creator
-    description += ' (forwarded request %d from %s)' % [params[:number], who]
 
-    req = nil
```

Comment:
It is similar to https://github.com/openSUSE/open-build-service/pull/2331/files#diff-d80ef30560c59a982e9ca5ff125427fbR505 :confused: